### PR TITLE
VIPER - async crossref calls for DOI filling

### PR DESCRIPTION
**Pull request by chreman**
* integrates rOpenSci rcrossref package
* identifies missing DOIs in publications
* tries to fill missing DOIs through asynchronous calls to Crossref
* uses title matching and a final sanity check via levenshtein edit distance with a max. of 5

### CODE REVIEW

File: doc/server_config.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Make sure you have the following packages installed:
   * jaod (Currently, a Github repository only: http://github.com/ropenscilabs/jaod. Install with  devtools.)
   * rbace (Currently, a Github repository only: http://github.com/ropenscilabs/rbace. Install with devtools.)
   * ropenaire (For VIPER. Currently, a Github repository only: https://github.com/sckott/ropenaire. Install with devtools.)
-  * readr (for ropenair/VIPER only)
+  * readr (for ropenair/VIPER)
+  * rcrossref (for VIPER)
```
Comment (pkraker) [inline doc/server_config.md]: Please add a description on how to install the async version (for now).

**Comment by pkraker** [comment]
Contrary to what I said above, the test case *does* work out of the box. It does, however, produce the following warning:

```
Warning in mapply(FUN = f, ..., SIMPLIFY = FALSE) :
  longer argument not a multiple of length of shorter
```

**Comment by chreman** [comment]
Added two commits two address utils.R loading and the rcrossref async install.

Unfortunately I cannot improve on your first comment, as the test works for me out-of-the-box. What is the error message you encounter?

**Comment by pkraker** [comment]
Thanks! Did a clean checkout and it works now out of the box. Merging this now.

**Pull request merged**